docs/architecture/managers/peers-manager.md

```diff
@@ -92,21 +92,28 @@ The way other actors will communicate with the storage manager is:
 
 These are the messages sent by the peers manager:
 
-| Message   | Destination   | Input type | Output type                 | Description               |
-| --------- | ------------- | ---------- | --------------------------- | ------------------------- |
-| GetConfig | ConfigManager | `()`       | `Result<Config, io::Error>` | Request the configuration |
-
-It is foreseen that in the future, the peers manager will call the `Storage Manager` to store a list of known peers.
+| Message   | Destination     | Input type                                | Output type                 | Description                               |
+| --------- | --------------- | ----------------------------------------- | --------------------------- | ----------------------------------------- |
+| GetConfig | ConfigManager   | `()`                                      | `Result<Config, io::Error>` | Request the configuration                 |
+| Put       | StorageManager  | `&'static [u8]`, `Vec<u8>`                | `StorageResult<()>`         | Wrapper to RocksStorage `put()` method    |
 
 #### GetConfig
 
 This message is sent to the [`ConfigManager`][config_manager] actor when the peers manager actor is started.
 
 The return value is used to initialize the list of known peers. For further information, see  [`ConfigManager`][config_manager].
 
+#### Put
+
+This message is sent to the [`StorageManager`][storage_manager] actor periodically using a period obtained from config file.
```

Comment:
The period is not necessarily taken from the config file as it can be overridden by CLI arguments. All this actor knows about is the `ConfigManager`.